zerver/worker/queue_processors.py

```diff
@@ -241,10 +241,10 @@ def consume(self, data: Mapping[str, Any]) -> None:
                 "zerver/emails/invitation_reminder",
                 referrer.realm,
                 to_emails=[invitee.email],
-                from_address=FromAddress.tokenized_no_reply_address(),
                 language=referrer.realm.default_language,
                 context=context,
-                delay=datetime.timedelta(days=settings.INVITATION_LINK_VALIDITY_DAYS - 2))
+                delay=datetime.timedelta(days=settings.INVITATION_LINK_VALIDITY_DAYS - 2),
+                sender=FromAddress.tokenized_no_reply_address)
```

Comment:
I think we probably want to just pass something like `sender=FromAddress.tokenized_no_reply_placeholder` with that having a value outside the namespace of valid email addresses, like "TOKENIZED_NOREPLY"`, that is replaced inside `send_email`.

Reply:
@timabbott I am sorry but I didn't quite understand it. Can you clear me out with this approach?

Reply:
The suggestion is to store a placeholder string in ScheduledEmail that `send_email` will convert into the tokenized noreply email just before sending.